Fixing typings

hyochan opened the pull request:
This boilerplate is not correctly typed and need to fix this issue.

hyochan committed:
Fixing typings

hyochan commented:
@devethan Could you kindly tell where I can find any doc related to `subscribeGrapqhQL.ts` file?

![image](https://user-images.githubusercontent.com/27461460/83652809-00ab8500-a5f6-11ea-936c-21db958d0d78.png)

I am currently looking up for above issue.

devethan commented:
> @devethan Could you kindly tell where I can find any doc related to `subscribeGrapqhQL.ts` file?
> 
> ![image](https://user-images.githubusercontent.com/27461460/83652809-00ab8500-a5f6-11ea-936c-21db958d0d78.png)
> 
> I am currently looking up for above issue.

I reffered below pages.
- https://www.howtographql.com/react-relay/7-subscriptions/
- https://github.com/howtographql/react-relay/blob/master/src/Environment.js

devethan commented:
@hyochan I made an issue about this. facebook/relay/issues/3091

devethan committed:
Fix type error in temporary

https://github.com/dooboolab/expo-relay-boilerplate/pull/2\#issuecomment-638253400

devethan committed:
Fix type error with Fragment and return type

devethan committed:
Modify babel config to fix resolve problem
